src/services/donationService.ts

```diff
@@ -198,9 +198,6 @@ export const updateTotalDonationsOfProject = async (
     `,
       [projectId],
     );
```

Comment:
this is a pretty common function, called even inside the parent functions like syncing donations with blockchain. So it was calling refreshProjectDonationSummaryView twice at least. 
And on every donation. It's best to remove from here and let the cronjob and other functions update the data